projects/mat-video/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "mat-video",
+  "description": "",
+  "version": "2.7.2",
+  "private": false,
+  "peerDependencies": {
```

Comment:
Will setting the below peer dependencies to >=8.0.0 break compatibility with previous releases?